ngrinder-runtime/src/main/java/org/ngrinder/sm/NGrinderLightSecurityManager.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ngrinder.sm;
+
+import java.net.InetAddress;
+
+/**
+ * nGrinder security manager.
```

Comment:
nGrinder security manager which less secured than NGrinderSecurityManager.

This allows followings.
* cmd execution
* multicast
* tcp connection unspecified address.